src/pymovements/utils/archives.py

```diff
@@ -138,6 +145,8 @@ def _extract_tar(
         source_path: Path,
         destination_path: Path,
         compression: str | None,
+        resume: bool,
```

Comment:
can you add a `*,` before `resume`? This is PR a breaking change either way, so we can make it more future proof.